packages/query/src/query/useList.ts

```diff
@@ -179,22 +180,22 @@ const withPostgrestFilters = () => (props: UseListProps & UseListFilters) => {
 
   const getPartitionedFilters = (
     key: string,
-    filters: Record<string, any>[]
+    filters: Record<string, any>[],
   ) => {
     // @example qs = brand=in.1&price=lt.500&brand=in.5;
     // filter = [{ key: 'price', op: 'lt', value: '500' }, { key: 'brand', op: 'in', value: '(1,5)' }]
 
     const [arrayBasedFilters, otherFilters] = partition(filters, (filter) =>
-      arrayBasedOps.includes(filter.op)
+      arrayBasedOps.includes(filter.op),
     )
 
     const groupedArrayBasedFilters = groupBy(
       arrayBasedFilters,
-      (filt) => filt.op
+      (filt) => filt.op,
```

Comment:
Able to check your IDE settings. These extra commas pollute the PR and are not part of our eslint rules.